deploy/clickhouse-keeper/clickhouse-keeper-1-node-256M-for-test-only.yaml

```diff
@@ -0,0 +1,246 @@
+---
+# Fake Service to drop-in replacement in tests
+apiVersion: v1
+kind: Service
+metadata:
+  # DNS would be like zookeeper.namespace.svc
+  name: zookeeper
+  labels:
+    app: zookeeper
+spec:
+  ports:
+    - port: 2181
+      name: client
+    - port: 7000
+      name: prometheus
+  selector:
+    app: clickhouse-keeper
+    what: node
+---
+# Setup Service to provide access to ClickHouse keeper for clients
+apiVersion: v1
+kind: Service
+metadata:
+  # DNS would be like clickhouse-keeper.namespace.svc
+  name: clickhouse-keeper
+  labels:
+    app: clickhouse-keeper
+spec:
+  ports:
+    - port: 2181
+      name: client
+    - port: 7000
+      name: prometheus
+  selector:
+    app: clickhouse-keeper
+    what: node
+---
+# Setup Headless Service for StatefulSet
+apiVersion: v1
+kind: Service
+metadata:
+  # DNS would be like keeper-0.clickhouse-keepers.namespace.svc
+  name: clickhouse-keepers
+  labels:
+    app: clickhouse-keeper
+spec:
+  ports:
+    - port: 9234
+      name: raft
+  clusterIP: None
+  selector:
+    app: clickhouse-keeper
+    what: node
+---
+# Setup max number of unavailable pods in StatefulSet
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: clickhouse-keeper-pod-disruption-budget
+spec:
+  selector:
+    matchLabels:
+      app: clickhouse-keeper
+  maxUnavailable: 1
+---
+# Setup ClickHouse Keeper settings
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: clickhouse-keeper-settings
+data:
+  keeper_config.xml: |
+    <yandex>
+        <include_from>/tmp/clickhouse-keeper/config.d/generated-keeper-settings.xml</include_from>
+        <logger>
+            <level>trace</level>
+            <console>true</console>
+        </logger>
+        <listen_host>0.0.0.0</listen_host>
+        <keeper_server incl="keeper_server">
+            <path>/var/lib/clickhouse-keeper</path>
+            <tcp_port>2181</tcp_port>
+            <coordination_settings>
+                <!-- <raft_logs_level>trace</raft_logs_level> -->
+                <raft_logs_level>debug</raft_logs_level>
```

Comment:
check also https://github.com/ClickHouse/ClickHouse/blob/bc2d0ee7c7c9be67825ae3f8acf8a9f08bc7c331/tests/config/config.d/keeper_port.xml#L17

Reply:
I checked, for `raft_logs_level debug` I see only changes in quorum + ruok/mntr execution command from liveness / readyness probes, for trace log 

I don't think we need to change anything else on this ConfigMap related with standalone mode execution